dm/tests/shardddl4/run.sh

```diff
@@ -31,7 +31,7 @@ function DM_119_CASE {
 	#         "query-status test" \
 	#         "because schema conflict detected" 1
 	# fi
-	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10 "fail"
+	# check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10 "fail"
```

Comment:
Prefer to add a TODO comment here.

Reply:
add it in https://github.com/pingcap/tiflow/pull/4031/commits/5458724ae9b9fa747794ff2d0ce4524ad94c2107. 

ptal